lib/booktype/apps/edit/static/edit/js/booktype/editor.js

```diff
@@ -76,6 +76,10 @@
             'TABLE': ['horizontalline', 'pbreak', 'indent-left', 'indent-right'],
             'PRE': ['insertImage', 'insertLink', 'horizontalline', 'pbreak', 'indent-left', 'indent-right'],
             'ALL': ['insertImage', 'insertLink', 'horizontalline', 'pbreak', 'indent-left', 'indent-right']
+          },
+          'autosave': {
+            'enabled': true,
+            'delay': 10
           }
```

Comment:
Hi @olegpshenichniy, 
only thing I would like to change is to have a place to easily deactivate the autosave functionality from instances. It could be on settings file or in book settings like what we have for icejs plugin. What do you think?